package.json

```diff
@@ -28,7 +28,7 @@
     "pre-commit": "lint-staged"
   },
   "dependencies": {
-    "@openstfoundation/mosaic.js": "0.10.0-beta.2"
+    "@openstfoundation/mosaic.js": "0.10.0-beta.4"
```

Comment:
why 4 not 3 ?

Reply:
I think 4 is the latest mosaic beta version. openst.js develop points to the beta.4